include/walker-dropdown.php

```diff
@@ -19,19 +19,24 @@ class PLL_Walker_Dropdown extends Walker {
 	public $db_fields = array( 'parent' => 'parent', 'id' => 'id' );
 
 	/**
-	 * Outputs one element
+	 * Outputs one element.
 	 *
 	 * @since 1.2
 	 *
-	 * @param string   $output            Passed by reference. Used to append additional content.
-	 * @param stdClass $element           The data object.
-	 * @param int      $depth             Depth of the item.
-	 * @param array    $args              An array of additional arguments.
-	 * @param int      $current_object_id ID of the current item.
+	 * @param string                $output            Passed by reference. Used to append additional content.
+	 * @param PLL_Language|stdClass $element           The data object. `PLL_Language` or `stdClass` in our case.
+	 * @param int                   $depth             Depth of the item.
+	 * @param array                 $args              An array of additional arguments.
+	 * @param int                   $current_object_id ID of the current item.
 	 * @return void
 	 */
 	public function start_el( &$output, $element, $depth = 0, $args = array(), $current_object_id = 0 ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
 		$value_type = $args['value'];
+
+		if ( $element instanceof PLL_Language ) {
+			$element = $element->to_standard_class();
+		}
+
```

Comment:
In fact, is the conversion useful here? I would think it's useful for `display_element()` only.

If it's useful, could we avoid doing this operation 2 times by moving the conversion to `walk()`?
```php
foreach ( $elements as &$element ) {
	if ( $element instanceof PLL_Language ) {
		$element = $element->to_standard_class();
	}
}
```

Reply:
It's not useful as is, but in my mind, since these are public method I wanted to handle as many cases as possible. But it may be overkill.
As you said, I leave the conversion in `display_element()` in 1064110.